superset/assets/src/dashboard/stylesheets/dashboard.less

```diff
@@ -63,7 +79,7 @@
   padding: 0 16px;
   position: absolute;
   top: 0;
-  right: -22px;
+  right: -16px;
```

Comment:
can you add a comment that you're doing this to increase the clickable area for the button?

Reply:
fixed.